connectors-config/connections/source.json

```diff
@@ -1,14 +1,13 @@
 {
   "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
-  "topic.prefix": "dbserver1",
+  "topic.prefix": "DBProduct",
   "database.user": "admin",
   "database.dbname": "product",
```

Comment:
can we filter the tables, I see at the moment all the kafka connect create 1 topic for every table

Reply:
I think we can exclude tables which don't need to be tracked, I've tried to exclude but still no success. I will find out more about this issue.